pep/app/pods/components/home/component.ts

```diff
@@ -142,4 +141,27 @@ export default class Home extends Component<HomeArgs> {
             term
         });
     }
+
+    /**
+     * Handle the resizing of the "Expert Pick of the Day" card.
+     *
+     * This does a more proper job of handling the resizing from the side-panels
+     * than bootstrap alone.
+     *
+     * This is also called on the insertion of the '.expert-pick-graphic-container' to ensure it's properly
+     * sized and spaced.
+     * @param el
+     */
+    @action
+    handleResize(el: HTMLElement) {
+        const graphicContainer = el.className === 'card-body' ? el.querySelector('.expert-pick-graphic-container') : el;
+        if (graphicContainer) {
+            if (el.clientWidth < 870) {
```

Comment:
Might be best to put this in a constant, or if possible re-use one of the already existing breakpoints from the media service

Reply:
@bakerac4 It might make sense as a property on the component itself I think if anything. That `870` comes from the approximate width of the expert-pick card where the graphic to the right of the expert-pick text pops up above it and its current styling was making it look pretty bad. (the border-left was there when it shouldn't be, margins were off, etc). It basically became more of a side-panel being open/closed issue than screen-size by itself. That's why I felt it was better to solve it this way rather than with bootstrap or mediaService breakpoints

Reply:
I'd be ok with making it a property on the component 👍🏻